services/hardware/src/routes/item.ts

```diff
@@ -135,6 +135,11 @@ itemRouter.route("/").post(
     if (!req.body.location.trim().length) {
       throw new BadRequestError("The location for this item can't be blank.");
     }
+    req.body.category = parseInt(req.body.category);
+    req.body.location = parseInt(req.body.location);
+    req.body.price = parseFloat(req.body.price);
+    req.body.totalAvailable = parseFloat(req.body.totalAvailable);
+    req.body.maxRequestQty = parseFloat(req.body.maxRequestQty);
```

Comment:
Its generally bad practice to modify the req.body directly since it can lead to situations later on where u don't know the exact value or type. Instead, either create a new variable or just adjust the usage where the variable is used. In this case, what you can do is override it like ive suggested below on line 166

```
...req.body,
category: parseInt(req.body.category),
```
etc.